README.md

```diff
@@ -43,6 +43,17 @@ Dependency: Python 3.9+ and pip3 installed. `sudo apt-get install python3-pip` i
 4. `pip3 install -r requirements.txt`
 5. `python3 main.py` or `./main.py`
 
+## Should I set the elevation?
+
+The barometric (= atmospheric) pressure is affected by the temperature, humidity and altitude (= elevation): while the standard barometric pressure at sea-level is 1013.25 mbar, it's 300 mbar at the top of Mount Everest. 
+
+For that reason, weather stations always adjust their recorded surface pressure to the standard [mean sea-level pressure](https://en.wikipedia.org/wiki/Atmospheric_pressure#Mean_sea-level_pressure) and that pressure is the one reported in weather reports on the radio, television, newspapers, official organizations such as [NOAA](https://www.wpc.ncep.noaa.gov/basicwx/92fndfd.gif), etc. Otherwise, comparing pressures recorded at two different locations would be akin to comparing apples to oranges.
+
+AirThings asks for the device location but doesn't use this information (nor the recorded temperature) to adjust its recorded surface pressure. Therefore if you have any other weather station (such as the [Davis Vantage](https://www.davisinstruments.com/pages/vantage-pro2)) or compare Airthing's values to official ones, you will get different readings, unless you live at sea-level and it's currently [15 degrees Celsius](https://en.wikipedia.org/wiki/Standard_sea-level_conditions).
```

Comment:
Have you considered contacting AirThings support with this info?  I guess you did, just checking.

Reply:
Yes, of course :)

My ticket:

> Hi,
The pressure sensor is off, and my understanding is that it doesn’t need a calibration period. Any ideas how to fix that?
The graph below is the pressure reported in the past 48h by:
•	A Davis Vantage pro weather station for Kitchen 
•	[iqair](https://www.iqair.com/) for Bellevue, Redmond, Seattle
•	Airthings for Bedroom 
Removing iqair to remove some noise shows near-identical trends for Davis and Airthings:
Diffing these graphs shows the difference is between 10.6 and 11.4 millibars

Their original answer:
> Thank you for contacting us here at Airthings!
This is a good observation, and I am happy to address your concern. First things first, based on the database, the monitor paired with your account is for consumer use and not a professional monitor. 
Keep in mind that several pressure monitors are built; if similar detectors are built and manufactured with the same sensors, the readings should be similar, or at least within the set standard deviation. 
When both units are in the same proper conditions, this is simple and achievable. 
However, if the devices are not under the same conditions, particularly if the monitors are of a different type, it will be difficult to draw any conclusions. 
The pressure graph is based on the database is stable and does not infer anything especially if you are comparing a monitor designed indoors vs a monitor for outdoors. 

My reply:
> Unless your house is completely airtight and you have something creating positive or negative pressure (e.g, some very powerful AC), [the barometric pressure](https://en.wikipedia.org/wiki/Barometric_formula) is going to be the same inside and outside. The Davis vantage pro is a consumer device that was bought 17 years ago, so not exactly best grade. 

> My guess is that although Airthings is asking for the location of the device, it is not using this information to get the elevation when computing the barometric pressure. My current elevation is 100m, which corresponds to 10mbars difference when compared to sea level, which is the difference between what’s reported by Airthings and what’s reported by the other sensors.


Their final answer was a 1-liner e-mail consisting of the below link that appears to have been created ... as a result of our e-mail exchange (given the post's timestamp) ;)


[Air pressure: why is it different from my weather report?](https://help.airthings.com/en/articles/3429386-air-pressure-why-is-it-different-from-my-weather-report)